Detections/ThreatIntelligenceIndicator/IPEntity_WireData.yaml

```diff
@@ -0,0 +1,5 @@
+id: 7f327be5-b165-443c-9500-e9a35fccc062
+name: TI map IP entity to WireData
+description: |
+    As part of content migration, this file is moved to new location. you can find here: https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Threat%20Intelligence'
```

Comment:
Hi, I didn't find any file with the name IPEntity_WireData in Threat Intelligence folder. Can you please help me with the new location. Thanks